.github/workflows/bot-tropic-emulator.yml

```
@@ -0,0 +1,87 @@
+            <!-- ui-comment-${{ github.workflow_ref }} -->
+          edit-mode: replace
+
+  issue_open:
+    name: Open an issue for trezor-user-env
+    runs-on: ubuntu-latest
+    #if: github.event.action == 'closed' && github.event.merged == false  # FIXME FIXME FIXME
+    env:
+      URL: ${{ github.event.pull_request.html_url }}
+    steps:
+      - name: Generate GitHub App token
+        if: false # FIXME FIXME FIXME
```

actionlint (1.7.12)

[error] constant expression "false" in condition. remove the if: section

(if-cond)